androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt

```diff
@@ -0,0 +1,90 @@
+package com.projectnewm.compose
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.navigation.NavController
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import com.projectnewm.R
+import kotlinx.coroutines.flow.collect
+
+@Composable
+internal fun NEWMApp(navController: NavHostController = rememberNavController()) {
+    Scaffold(
+        bottomBar = {
+            val currentRootScreen by navController.currentRootScreenAsState()
+            NEWMBottomNavigation(
+                currentRootScreen = currentRootScreen,
+                onNavigationSelected = { navController.navigate(it.route) }
+            )
+        }
+    ) { }
+}
+
+@Composable
+internal fun NEWMBottomNavigation(
+    currentRootScreen: Screen,
+    onNavigationSelected: (Screen) -> Unit
+) {
+    BottomNavigation {
+        HomeBottomNavigationItem(
+            icon = R.drawable.ic_home_black_24dp,
+            selected = currentRootScreen == Screen.HomeRoot,
+            label = stringResource(R.string.home),
+            onClick = { onNavigationSelected(Screen.HomeRoot) }
+        )
+        HomeBottomNavigationItem(
+            icon = R.drawable.ic_dashboard_black_24dp,
+            selected = currentRootScreen == Screen.TribeRoot,
+            label = stringResource(R.string.tribe),
+            onClick = { onNavigationSelected(Screen.TribeRoot) }
+        )
+        HomeBottomNavigationItem(
+            icon = R.drawable.ic_notifications_black_24dp,
```

Comment:
we should be able to get the proper icons now but we can do that as a follow up

Reply:
I added the icons on the last commit, however they seem to need a bit of work from design, they don't scale well on the app on their current form